front/pages/w/[wId]/workspace/index.tsx

```diff
@@ -415,7 +415,7 @@ function ContentCreationSharingToggle({ owner }: { owner: WorkspaceType }) {
       <div className="h-full border-b border-border dark:border-border-night" />
       <ContextItem
         title="Content Creation file sharing"
-        subElement="Enable Content Creation files sharing"
+        subElement="Enable sharing Content Creation files with the public"
```

Comment:
Same here, enable works but maybe allow is more appropriate? 
```suggestion
        subElement="Allow public sharing of Content Creation files"
```